audit.sql

```diff
@@ -239,3 +239,14 @@ $body$ LANGUAGE 'sql';
 COMMENT ON FUNCTION audit.audit_table(regclass) IS $body$
 Add auditing support to the given table. Row-level changes will be logged with full client query text. No cols are ignored.
 $body$;
+
+CREATE OR REPLACE VIEW audit.tableslist AS 
+ SELECT DISTINCT triggers.trigger_schema AS schema,
+    triggers.event_object_table AS auditedtable
+   FROM information_schema.triggers
+  WHERE triggers.trigger_name::text = 'audit_trigger_row'::text
```

Comment:
This may only work for tables with row level tiggers
Could add a "OR ... = 'audit_trigger_stm' " for completenesss